modules/local/build_intervals/main.nf

```diff
@@ -1,22 +1,30 @@
 process BUILD_INTERVALS {
-    tag "$fasta_fai"
+    tag "$meta.id"
 
-    conda (params.enable_conda ? "anaconda::gawk=5.1.0" : null)
+    conda (params.enable_conda ? "bioconda::gawk=4.1.3" : null)
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/gawk:5.1.0' :
-        'quay.io/biocontainers/gawk:5.1.0' }"
+        'https://depot.galaxyproject.org/singularity/gawk:4.1.3--0':
+        'quay.io/biocontainers/gawk:4.1.3--1' }"
```

Comment:
Why did we went to an older version?

Reply:
oh I think this is a leftover from when I tried to get it to work with the cutandrun example